src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs

```diff
@@ -1545,7 +1545,6 @@ public override void VisitFlowCapture(IFlowCaptureOperation operation)
                     break;
 
                 case OperationKind.OmittedArgument:
-                case OperationKind.DeclarationExpression:
```

Comment:
Because, in the case of interpolated string handlers, the declaration expression is actually captured. See the graph in the test case I added, which would fail here because the declaration expression goes through here.